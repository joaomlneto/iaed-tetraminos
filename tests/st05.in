16
2 3 R
2 5 R
1 7 R
1 7 R
1 3 R
1 3 R
2 7 R
2 9 R
2 3 R
2 5 R
1 7 R
1 7 R
1 3 R
1 7 R
2 1 R
2 1 R
