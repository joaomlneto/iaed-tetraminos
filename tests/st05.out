1
12

|  RRRRRRRR| 1
 ----------
